src/test/regress/bin/create_test.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+
+import sys
+import random
+import os
+
+if len(sys.argv) != 2:
+    raise Exception("Expected the name of the new test as an argument")
+
+test_name = sys.argv[1]
+
+filename = f"src/test/regress/sql/{test_name}.sql"
```

Comment:
So this assumes that we are running the script from the top directory. I think it would be useful if the script was generating the file no matter where you run it from.